lib/SCSI2SD/include/scsi2sd.h

```diff
@@ -83,7 +83,8 @@ typedef enum
 	S2S_CFG_QUIRKS_APPLE = 1,
 	S2S_CFG_QUIRKS_OMTI = 2,
 	S2S_CFG_QUIRKS_XEBEC = 4,
-	S2S_CFG_QUIRKS_VMS = 8
+	S2S_CFG_QUIRKS_VMS = 8,
+	S2S_CFG_QUIRKS_EMU = 9
```

Comment:
I wonder if this feature could be generally useful also - if so, it would be better to use a separate setting like `NameFromImage = 1` instead of tying it to the quirks setting.